source/theory.rst

```diff
@@ -114,6 +124,10 @@ where :math:`\Re` denotes the real part of the formula, :math:`R_{f}` is the ram
 
 Convolution Integral Formulation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. Note:: 
+	Adam:
+	Only a small portion of this section is about the CI form. Maybe better named as 'Irregular Response' or 'Unsteady Response' to contrast with the 'Sinusoidal Steady-State Response' heading. Below, add something like 'in the case of an irregular wave spectrum, the fluid memory has an important impact on the WEC dynamics. This fluid memory effect is captured by the convolution integral formulation based upon...'
+
```

Comment:
Generally I agree but I think this also relates to what the Theory section is intended to do. What you've suggested is somewhere between theory and how it's implemented/used in WEC-Sim. Do you have a suggestion on how to best differentiate between those topics? It's a common issue in the Theory section.